src/main/java/com/github/fishio/control/SinglePlayerController.java

```diff
@@ -258,9 +258,6 @@ public void revive() {
 		ICollisionArea area = pf.getStartCollisionArea();
 		player.setBoundingArea(area);
 		
-		//Reset the speed of the fish.
-		player.setSpeedVector(new Vec2d(0, 0));
-		
```

Comment:
Good of you to notice. I temporary removed that line of code because of annoying compile errors, but I forgot to re-add that bugfix again. I'll try and fix that.